site/src/Librecores/ProjectRepoBundle/Entity/ProjectRelease.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+namespace Librecores\ProjectRepoBundle\Entity;
+
+class ProjectRelease
+{
+    /**
+     * Name of the release
+     *
+     * @var string
+     */
+    private $name;
+
+    /**
+     * Date when release was published
+     *
+     * @var \DateTime
+     */
+    private $publishedAt;
+
+    /**
+     * Is the release a pre-release
+     *
+     * @var bool
+     */
+    private $isPrerelease;
+
+    /**
+     * Commit ID this release is based on
+     *
+     * @var string
```

Comment:
The idea is to use this class also for releases which are not derived from the source repo, yes? In this case, commit id might not be available. Make it nullable.